.rubocop.yml

```diff
@@ -95,7 +95,7 @@ Metrics/BlockNesting:
   Max: 4
 
 Metrics/ClassLength:
-  Max: 575
+  Max: 585
```

Comment:
Let's disable the cop in conference model all together, it seems to be a regular issue.
https://github.com/openSUSE/osem/pull/1118/files#diff-a8447a840df703618503bdc301e044e9R1
https://github.com/openSUSE/osem/pull/1118/files#diff-a8447a840df703618503bdc301e044e9R1067